automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/FireCloudClient.scala

```diff
@@ -62,6 +62,14 @@ trait FireCloudClient {
     }
   }
 
+  import scala.reflect.ClassTag
+  import scala.reflect._
+  def parseResponseAs[T: ClassTag](response: HttpResponse): T = {
```

Comment:
Parses simple JSON case classes

TODO: Jackson equivalents of our Spray formatters